browser/components/MarkdownPreview.js

```diff
@@ -127,13 +127,21 @@ h1, h2 {
   border: none;
 }
 
+h3 {
+  margin: 1em 0 0.4em;
+}
+
+h4, h5, h6 {
+  margin: 1em 0 0.5em;
```

Comment:
Can you change this to `1.1em`? otherwise `h5` will disappear on 80%
```suggestion
  margin: 1.1em 0 0.5em;
```